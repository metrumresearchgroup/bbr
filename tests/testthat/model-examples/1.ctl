$PROBLEM PK model 1 cmt base

$INPUT ID TIME MDV EVID DV AMT  SEX WT ETN
$DATA ../../data/acop.csv IGNORE=@
$SUBROUTINES ADVAN2 TRANS2

$PK
ET=1
IF(ETN.EQ.3) ET=1.3
KA = THETA(1)
CL = THETA(2)*((WT/70)**0.75)* EXP(ETA(1))
V = THETA(3)*EXP(ETA(2))
SC=V


$THETA
(0, 2)  ; KA
(0, 3)  ; CL
(0, 10) ; V2
(0.02)  ; RUVp
(1)     ; RUVa

$OMEGA
0.05    ; iiv CL
0.2     ; iiv V2

$SIGMA
1 FIX

$ERROR
IPRED = F
IRES = DV-IPRED
W = IPRED*THETA(4) + THETA(5)
IF (W.EQ.0) W = 1
IWRES = IRES/W
Y= IPRED+W*ERR(1)

$EST METHOD=1 INTERACTION MAXEVAL=9999 SIG=3 PRINT=5 NOABORT POSTHOC
$COV
