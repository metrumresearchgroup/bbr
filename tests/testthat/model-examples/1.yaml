description: original acop model
model_type: nonmem
tags:
- acop tag
- other tag
bbi_args:
  overwrite: yes
  threads: 4.0
