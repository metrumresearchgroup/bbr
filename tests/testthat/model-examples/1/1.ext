TABLE NO.     1: First Order Conditional Estimation with Interaction: Goal Function=MINIMUM VALUE OF OBJECTIVE FUNCTION: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0
 ITERATION    THETA1       THETA2       THETA3       THETA4       THETA5       SIGMA(1,1)   OMEGA(1,1)   OMEGA(2,1)   OMEGA(2,2)   OBJ
            0  2.00000E+00  3.00000E+00  1.00000E+01  2.00000E-02  1.00000E+00  1.00000E+00  5.00000E-02  0.00000E+00  2.00000E-01    15294.002144233427
            5  4.24454E+00  2.01268E+01  1.84447E+01 -6.04580E-03  5.26539E+00  1.00000E+00  8.69929E-02  0.00000E+00  8.29972E-01    3740.5412530895987
           10  5.00791E+00  7.71104E+01  5.03064E+02 -1.32147E-01  6.25451E+00  1.00000E+00  7.08461E-03  0.00000E+00  1.59955E-01    3133.0637805488759
           15  2.40082E+00  5.37872E+01  3.82758E+02 -7.69871E-02  4.03637E+00  1.00000E+00  1.16655E-01  0.00000E+00  2.43800E-01    2648.7566483156047
           20  2.30528E+00  5.58065E+01  4.72255E+02 -8.05144E-02  4.12591E+00  1.00000E+00  9.09862E-02  0.00000E+00  2.06974E-01    2638.5029836424560
           25  2.31302E+00  5.44961E+01  4.65370E+02 -8.07920E-02  4.13612E+00  1.00000E+00  9.61335E-02  0.00000E+00  1.57375E-01    2636.8778706340345
           28  2.31034E+00  5.49596E+01  4.64659E+02 -8.05722E-02  4.13030E+00  1.00000E+00  9.64407E-02  0.00000E+00  1.53571E-01    2636.8457703290751
  -1000000000  2.31034E+00  5.49596E+01  4.64659E+02 -8.05722E-02  4.13030E+00  1.00000E+00  9.64407E-02  0.00000E+00  1.53571E-01    2636.8457703290751
  -1000000001  8.61470E-02  3.32914E+00  2.96177E+01  5.55149E-02  1.35989E+00  1.00000E+10  2.00146E-02  1.00000E+10  2.67330E-02    0.0000000000000000
  -1000000004  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  1.00000E+00  3.10549E-01  0.00000E+00  3.91882E-01    0.0000000000000000
  -1000000005  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  1.00000E+10  3.22245E-02  1.00000E+10  3.41085E-02    0.0000000000000000
  -1000000006  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  1.00000E+00  0.00000E+00  1.00000E+00  0.00000E+00    0.0000000000000000
  -1000000007  0.00000E+00  3.70000E+01  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00    0.0000000000000000
  -1000000008 -3.39427E-03  1.00007E-04 -7.27007E-04 -1.46088E-01  2.43350E-03  0.00000E+00  4.84108E-02  0.00000E+00 -5.58021E-03    0.0000000000000000
