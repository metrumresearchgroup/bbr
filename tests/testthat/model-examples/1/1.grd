TABLE NO.     1: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0
 ITERATION    GRD(1)       GRD(2)       GRD(3)       GRD(4)       GRD(5)       GRD(6)       GRD(7)
            0 -1.71398E+03 -3.12900E+03 -1.51232E+03 -9.72381E+03 -1.26854E+05 -8.15523E+03 -5.92268E+03
            5  1.17078E+02 -3.15649E+02 -3.12087E+02  4.92056E+02  1.16084E+03 -1.56290E+02 -9.98737E+02
           10  5.00463E+02  6.54364E+02  5.51559E+01 -2.56911E+02 -6.63058E+01 -6.29818E+01  1.86583E+01
           15  6.77246E+01 -4.43219E+00 -6.48126E+01 -9.39446E+00 -1.16784E+02  8.16779E+00  1.57055E+01
           20  1.65284E+00  8.79720E+00  5.48428E+00 -3.89023E+00 -9.06470E+00 -2.71057E+00  1.90967E+01
           25  5.40777E-02 -4.33555E+00  2.29074E-01  5.50463E-01  5.83533E+00 -3.26574E-01  1.77026E+00
           28  1.55432E-02 -2.09761E-02 -1.45120E-01  5.82595E-02 -5.33290E-02 -1.88565E-02  3.40971E-03
