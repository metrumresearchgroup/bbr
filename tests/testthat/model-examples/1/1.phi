TABLE NO.     1: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0
 SUBJECT_NO   ID           ETA(1)       ETA(2)       ETC(1,1)     ETC(2,1)     ETC(2,2)     OBJ
            1            1  0.00000E+00  0.00000E+00  9.64407E-02  0.00000E+00  1.53571E-01    319.60680461982184
            2            2  1.18968E-01  1.38830E-01  3.42754E-02 -4.86489E-03  4.27048E-03    53.374266964924203
            3            3 -2.52285E-01 -2.39666E-01  1.74389E-02 -2.20969E-03  1.22459E-03    54.353165319165640
            4            4  1.48765E-01  3.09026E-01  4.38827E-02 -5.75983E-03  6.03120E-03    55.077454865613092
            5            5 -5.52762E-01 -6.81036E-02  2.57807E-02 -3.01699E-03  1.89823E-03    66.870226302922120
            6            6  3.12276E-01 -6.08649E-01  6.30470E-03 -2.13919E-03  1.75490E-03    72.399049752203041
            7            7 -2.43575E-01  1.15642E-01  3.34915E-02 -4.43708E-03  3.69174E-03    62.283782320300098
            8            8  1.38618E-02  4.21414E-01  5.07953E-02 -6.28720E-03  7.42140E-03    55.561217642024801
            9            9 -4.24683E-01 -2.43821E-01  1.72936E-02 -2.18452E-03  1.20328E-03    58.876444849020963
           10           10  1.07683E-02  3.60512E-01  4.91556E-02 -5.71950E-03  5.92832E-03    58.795796180294815
           11           11  2.55954E-01 -4.00614E-01  1.19059E-02 -3.52711E-03  2.70126E-03    52.915998309505660
           12           12 -2.19643E-01  2.40332E-01  4.02802E-02 -5.24390E-03  5.01562E-03    57.237543036106679
           13           13 -4.49624E-02  6.24586E-01  6.64925E-02 -6.94617E-03  9.81260E-03    58.567640584205172
           14           14  2.44521E-01  5.68453E-01  5.95358E-02 -6.96924E-03  9.86626E-03    59.515459180973096
           15           15 -1.72627E-01  2.32500E-01  4.08436E-02 -4.99946E-03  4.56382E-03    55.084060627975084
           16           16 -9.53119E-02 -1.36281E-01  2.19439E-02 -2.70234E-03  1.64243E-03    58.943851002712726
           17           17  3.37674E-02  3.25164E-01  4.62993E-02 -5.56962E-03  5.65262E-03    54.678253804930705
           18           18 -3.48064E-01  3.01605E-01  4.97305E-02 -5.25212E-03  4.58294E-03    58.679084393760668
           19           19 -2.29873E-01 -5.85155E-01  7.32942E-03 -1.51973E-03  7.28503E-04    61.665629237290105
           20           20  2.56306E-01  1.05513E-01  3.25368E-02 -5.34592E-03  4.68118E-03    57.412688051250456
           21           21 -3.29892E-02  3.16921E-02  2.87852E-02 -4.41756E-03  3.51835E-03    53.762100714605481
           22           22  2.26967E-01 -1.07789E-01  2.29180E-02 -4.48474E-03  3.29206E-03    55.866644263513940
           23           23 -7.69146E-03 -1.27381E-01  2.16010E-02 -3.47091E-03  2.36759E-03    53.325934162036688
           24           24  3.29886E-01 -4.45761E-01  1.08105E-02 -2.94838E-03  1.99833E-03    59.018399987078446
           25           25  1.84796E-01 -5.64641E-01  7.76943E-03 -2.07014E-03  1.25470E-03    60.283891548767350
           26           26  3.05879E-01 -2.78936E-01  1.63060E-02 -4.22672E-03  3.10141E-03    60.254810249547916
           27           27 -2.78466E-02  3.22517E-01  4.50485E-02 -5.72480E-03  6.00067E-03    57.971634529026389
           28           28 -1.69955E-01 -2.36171E-02  2.64590E-02 -3.55600E-03  2.55267E-03    54.899863779267200
           29           29  3.74218E-01 -5.02298E-01  9.06266E-03 -2.71323E-03  1.96563E-03    61.668329328021414
           30           30 -2.90441E-01 -3.47740E-01  1.33696E-02 -1.88821E-03  9.76073E-04    64.916924369615941
           31           31  5.56621E-01 -8.58538E-01  2.57737E-03 -9.04876E-04  6.77660E-04    61.548577641677326
           32           32  6.80898E-02  1.73921E-01  3.61215E-02 -5.10372E-03  4.65916E-03    59.884144310425931
           33           33  9.06286E-02  6.96141E-01  6.83240E-02 -7.30020E-03  1.18879E-02    57.211554050784223
           34           34  4.83903E-02  2.69321E-01  4.29908E-02 -5.22939E-03  4.97686E-03    52.805521279169660
           35           35  4.50955E-01 -3.45665E-01  1.39049E-02 -3.76640E-03  2.72884E-03    59.274093212449358
           36           36 -1.92633E-01  4.32114E-01  5.73479E-02 -6.00308E-03  6.24674E-03    59.052140249223413
           37           37 -2.67959E-01  1.81481E-01  4.06905E-02 -4.51942E-03  3.56710E-03    56.368194348711285
           38           38  8.53460E-02  4.75103E-01  5.52100E-02 -6.41540E-03  7.82429E-03    59.253916158711817
           39           39 -4.11953E-01 -3.33172E-01  1.39524E-02 -2.32353E-03  1.29634E-03    95.325058984653253
           40           40 -5.98422E-02 -4.03636E-01  1.18397E-02 -2.07331E-03  1.09478E-03    62.255620116787306
