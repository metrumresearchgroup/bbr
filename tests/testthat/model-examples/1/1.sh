#!/bin/bash

#$ -wd /data/home/sethg/rbabylon/tests/testthat/model-examples/1

/opt/NONMEM/nm74gf/run/nmfe74 1.ctl  1.lst 
