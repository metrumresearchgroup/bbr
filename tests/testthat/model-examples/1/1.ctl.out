Starting NMTRAN
  
 WARNINGS AND ERRORS (IF ANY) FOR PROBLEM    1
             
 (WARNING  2) NM-TRAN INFERS THAT THE DATA ARE POPULATION.
Recompiling certain components
Building NONMEM Executable
Starting nonmem execution ...
License Registered to: Metrum Research Group
Expiration Date:    14 JUL 2020
Current Date:       20 MAY 2020
Days until program expires :  54


 First Order Conditional Estimation with Interaction
 MONITORING OF SEARCH:


0ITERATION NO.:    0    OBJECTIVE VALUE:   15294.0021442334        NO. OF FUNC. EVALS.:   7
 CUMULATIVE NO. OF FUNC. EVALS.:        7
 NPARAMETR:  2.0000E+00  3.0000E+00  1.0000E+01  2.0000E-02  1.0000E+00  5.0000E-02  2.0000E-01
 PARAMETER:  1.0000E-01  1.0000E-01  1.0000E-01  1.0000E-01  1.0000E-01  1.0000E-01  1.0000E-01
 GRADIENT:  -1.7140E+03 -3.1290E+03 -1.5123E+03 -9.7238E+03 -1.2685E+05 -8.1552E+03 -5.9227E+03

0ITERATION NO.:    5    OBJECTIVE VALUE:   3740.54125308960        NO. OF FUNC. EVALS.:  43
 CUMULATIVE NO. OF FUNC. EVALS.:       50
 NPARAMETR:  4.2445E+00  2.0127E+01  1.8445E+01 -6.0458E-03  5.2654E+00  8.6993E-02  8.2997E-01
 PARAMETER:  8.5249E-01  2.0034E+00  7.1219E-01 -3.0229E-02  5.2654E-01  3.7690E-01  8.1154E-01
 GRADIENT:   1.1708E+02 -3.1565E+02 -3.1209E+02  4.9206E+02  1.1608E+03 -1.5629E+02 -9.9874E+02

0ITERATION NO.:   10    OBJECTIVE VALUE:   3133.06378054888        NO. OF FUNC. EVALS.:  42
 CUMULATIVE NO. OF FUNC. EVALS.:       92
 NPARAMETR:  5.0079E+00  7.7110E+01  5.0306E+02 -1.3215E-01  6.2545E+00  7.0846E-03  1.5996E-01
 PARAMETER:  1.0179E+00  3.3466E+00  4.0181E+00 -6.6074E-01  6.2545E-01 -8.7705E-01 -1.1712E-02
 GRADIENT:   5.0046E+02  6.5436E+02  5.5156E+01 -2.5691E+02 -6.6306E+01 -6.2982E+01  1.8658E+01

0ITERATION NO.:   15    OBJECTIVE VALUE:   2648.75664831560        NO. OF FUNC. EVALS.:  57
 CUMULATIVE NO. OF FUNC. EVALS.:      149
 NPARAMETR:  2.4008E+00  5.3787E+01  3.8276E+02 -7.6987E-02  4.0364E+00  1.1666E-01  2.4380E-01
 PARAMETER:  2.8266E-01  2.9864E+00  3.7448E+00 -3.8494E-01  4.0364E-01  5.2360E-01  1.9902E-01
 GRADIENT:   6.7719E+01 -4.4374E+00 -6.4806E+01 -9.1829E+00 -1.1642E+02  8.1714E+00  1.5709E+01

0ITERATION NO.:   20    OBJECTIVE VALUE:   2638.50298364246        NO. OF FUNC. EVALS.:  87
 CUMULATIVE NO. OF FUNC. EVALS.:      236
 NPARAMETR:  2.3053E+00  5.5806E+01  4.7226E+02 -8.0514E-02  4.1259E+00  9.0986E-02  2.0697E-01
 PARAMETER:  2.4206E-01  3.0233E+00  3.9549E+00 -4.0257E-01  4.1259E-01  3.9934E-01  1.1714E-01
 GRADIENT:   1.6516E+00  8.7974E+00  5.4827E+00 -3.8974E+00 -9.0988E+00 -2.7136E+00  1.9096E+01

0ITERATION NO.:   25    OBJECTIVE VALUE:   2636.87787063403        NO. OF FUNC. EVALS.:  86
 CUMULATIVE NO. OF FUNC. EVALS.:      322
 NPARAMETR:  2.3130E+00  5.4496E+01  4.6537E+02 -8.0792E-02  4.1361E+00  9.6133E-02  1.5738E-01
 PARAMETER:  2.4540E-01  2.9995E+00  3.9402E+00 -4.0396E-01  4.1361E-01  4.2686E-01 -1.9843E-02
 GRADIENT:   5.4693E-02 -4.3216E+00  2.2446E-01  5.4546E-01  5.8849E+00 -3.2360E-01  1.7547E+00

0ITERATION NO.:   28    OBJECTIVE VALUE:   2636.84577032908        NO. OF FUNC. EVALS.:  44
 CUMULATIVE NO. OF FUNC. EVALS.:      366
 NPARAMETR:  2.3103E+00  5.4960E+01  4.6466E+02 -8.0572E-02  4.1303E+00  9.6441E-02  1.5357E-01
 PARAMETER:  2.4425E-01  3.0080E+00  3.9387E+00 -4.0286E-01  4.1303E-01  4.2845E-01 -3.2076E-02
 GRADIENT:   1.5687E-02 -2.0836E-02 -1.4475E-01  5.7882E-02 -5.1950E-02 -1.8675E-02  3.4278E-03
 Elapsed estimation  time in seconds:     2.00
 Elapsed covariance  time in seconds:     0.56
 Elapsed postprocess time in seconds:     0.00
 Elapsed finaloutput time in seconds:     0.11
Done with nonmem execution
